### DISCUSSION TITLE
MIDI2LR stops sending codes to Lightroom

### DISCUSSION THREAD

**Discussion by PS-photography** [rsjaffe/MIDI2LR#754] [Problems]
Hi - I'm running Windows 11, Lightroom v13,  MIDI2LR 5.4.0.0
I'm not sure when this starts happening
When I reboot, all is well for some time. I go in and out of Lightroom for a while using my Behringer XTouch Mini - all ok
The after a while I try to use it and nothing happens when I press buttons and turn dials. If I reboot all is well... for a while
Its happening repeatedly
any ideas or workarounds?
The MIDI2LR says its Connected to Lightroom in Green color.

thank you

**Comment by antzpantz** [comment]
1) File > Plug-in Extras... > Close Application then Start Application and
try your controller buttons.
or.
2) Disconnect and reconnect the USB cable for your controller and repeat
Step 1.
or.
3) Repeat Step 2 but restart Lightroom.

You may have an issue with your USB cable or PC's USB plug.

**Reply by PS-photography** [reply]
Thank you antzpantz! I replaced the cable, started working - I'll see how it goes
